database/migrations/2020_06_24_000000_move_group_type_id_on_campaigns_table.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class moveGroupTypeIdOnCampaignsTable extends Migration
+{
+    /**
+     * Run the migration to recreate group_type_id column in correct position.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        $tableName = 'campaigns';
+        $columnName = 'group_type_id';
+
+        // Drop foreign index and the column.
+        Schema::table($tableName, function ($table) use ($columnName) {
+            $table->dropForeign([$columnName]);
+            $table->dropColumn($columnName);
+        });
+
+        // Recreate the column in the correct position.
+        Schema::table($tableName, function (Blueprint $table) use ($columnName) {
+            $table->unsignedInteger($columnName)->after('secondary_causes')->nullable()->index()->comment('The group type id the group is for.');
+            $table->foreign($columnName)->references('id')->on('group_types');
+        });
+    }
+
+    /**
+     * Reverse the migrations.
+     *
+     * @return void
+     */
+    public function down()
+    {
+        return;
```

Comment:
I take it the inverse of this switcharoo isn't applicable?

Reply:
It just seemed superfluous to add the inverse, which would entail dropping the column (losing the data again), and then adding it to the end of the table -- where I can't imagine we'd think ahh we really needed this field all the way at the end after `deleted_at`.

Reply:
LOL well when throwing fields at the end of the row becomes in vogue again, you'll be sorry.